template.rb

```diff
@@ -71,7 +71,10 @@ def apply_template!
   copy_file "simplecov", ".simplecov"
 
   copy_file "Procfile"
-  copy_file ".nvmrc"
+
+  copy_file "variants/backend-base/.node-version", ".node-version"
+  create_link ".nvmrc", ".node-version"
```

Comment:
I'm not really a fan of having symlinks in git repos as their support tends to be very unpredictable (especially when you add docker into the mix). I know that at least WSL can get fight-y with Windows because while both support symlinks now, they're not 1:1 so each OS layer views them as something different.

I'd rather we just copied `.nvmrc` to have the same contents as `.node-version` - we could stick a check in `bin/setup` to help guard against them getting out of date, as that would ensure new developers coming on projects wouldn't get bitten if node is updated (and others would see the PR)